### PR TITLE
add utility pallet into runtime

### PR DESCRIPTION
**Pull request by cuteolaf**
To enable batch transactions on the BitTensor network, I've added the `utility` pallet into the runtime.

### CODE REVIEW

File: runtime/src/lib.rs
Comment (Rubberbandits) [inline runtime/src/lib.rs:1]: Please leave the tabulation as is for the majority of the file

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -80,42 +81,42 @@ pub type Hash = sp_core::H256;
...
-	state_version: 1,
+    spec_name: create_runtime_str!("node-subtensor"),
+    impl_name: create_runtime_str!("node-subtensor"),
+    authoring_version: 1,
+    // The version of the runtime specification. A full node will not attempt to use its native
+    //   runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
+    //   `spec_version`, and `authoring_version` are the same between Wasm and native.
+    // This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
+    //   the compatible custom types.
+    spec_version: 119,
```
Comment (Rubberbandits) [inline runtime/src/lib.rs]: spec_version needs to be +1 for chain upgrades

**Review by Rubberbandits — CHANGES_REQUESTED** [review]

**Comment by camfairchild** [comment]
What steps have been done to test the integration?

File: runtime/src/lib.rs (hunk 1 above)
Comment (Rubberbandits) [reply runtime/src/lib.rs]: Not resolved

**Review by Rubberbandits — CHANGES_REQUESTED** [review]

**Comment by cuteolaf** [comment]
> What steps have been done to test the integration?

https://github.com/opentensor/subtensor/pull/125/commits/218ee7430735dbb3145d0e9a1b983a96fe7c1f07

**Review by Rubberbandits — APPROVED** [review]
LGTM

**Review by camfairchild — APPROVED** [review]
LGTM. 
Thanks for the contribution 😁

**Comment by shibshib** [comment]
Merged to mainnet @block 315,568 and testnet @block 267,113

**Pull request merged**